third_party/move/move-compiler-v2/tools/testdiff/src/main.rs

```diff
@@ -201,8 +187,11 @@ fn collect_tests(root: &str) -> anyhow::Result<BTreeSet<PathBuf>> {
     let mut result = BTreeSet::new();
     for entry in WalkDir::new(root) {
         let path = entry?.into_path();
-        if path.display().to_string().ends_with(".move") {
+        let path_str = path.display().to_string();
+        if path_str.ends_with(".move") {
             result.insert(path);
+        } else if path_str.ends_with(".unit_test") {
```

Comment:
Maybe worth documenting this with a comment (like you have in the PR description)?

Reply:
On the other hand, I see that there is indeed some documentation below, so this is like not needed.